deploy/serverpc/dlc/run_dlc.py

```diff
@@ -10,6 +11,6 @@
     args = parser.parse_args()
 
     file_mp4 = Path(args.file_mp4)
-    path_dlc = download_weights(version=__version__)
```

Comment:
`version` args defaults to `__version__` anyways, so this is unnecessary; furthermore, the versioning system for DLC will change in the future, and will no longer be tied to `iblvideo.__version__`